library/src/test/java/ch/decent/sdk/Scratchpad.kt

```diff
@@ -281,5 +284,18 @@ class Scratchpad {
 //    b.BE().readUint16(2).should.equal(refNum);
   }
 
+  @Test fun `custom operation`() {
+    val logger = LoggerFactory.getLogger("LOG")
+    val api = DCoreSdk.createForWebSocket(Helpers.client(logger), Helpers.wsUrl, logger)
+
+    val op = CustomOperation(4, Helpers.account, listOf(Helpers.account), "hello data".toByteArray().hex())
+    api.broadcastApi.broadcastWithCallback(Helpers.private, op)
+        .testCheck()
+  }
+
+  @Test fun `check hex`() {
```

Comment:
why is this test useful?

Reply:
this file is excluded from the tests
just some internal stuff here

Reply:
I still don't understand why this change is useful.